README.md

```diff
@@ -1,36 +1,39 @@
 # book
 
-LinuxBoot book which contains the documentation in markdown format
+Documentation for LinuxBoot
 
 ## Table of Contents
 
-1. Introduction
-2. Getting Started in 20mins
-  * apt-get install qemu ovmf
-  * git clone github.com/linuxboot/linuxboot
-  * Download existing Linux bzImage
-  * git clone UEFIReplace
-  * Shell replacement
-3. LinuxBoot
-4. Minimal Linux kernel
-5. The Initramfs:
-  * HEADS
-  * NERF
-6. UEFI
-  * EDKII
-  * OVMF
-7. Tools:
-  * UEFITool and UEFIReplace
-  * [UEFI Tool Kit (Ryan O'Leary and Gan Shun)](7b.UEFI_Tool_Kit/README.md)
-  * LinuxBoot Scripts (Trammell Hudson)
-  * FMAP tool
-8. TPMs
-9. SecureBoot and BootGuard
-10. u-root
-  * [U-root: A Go-based, Firmware Embeddable Root File System](10.u-root/README.md)
-11. [coreboot, u-root and systemboot](11.coreboot.u-root.systemboot/README.md)
-12. Arm + u-boot
-13. DUT, a simple Device Under Test framework [DUT, Device Under Test framework] (13.dut/README.md)
-14. CPU, an ssh style command which lets you see your files from anywhere. [CPU, wherever you go, there your files are] (14.cpu/README.md)
-15. [Case Studies](case_studies/README.md)
-15. Conclusion
+| #   | Chapter |
```

Comment:
Maybe we could bold chapter names and leave subsections as normal text so that we don't have to use the '--->' to indicate subsections?

Reply:
Please do!